sdk/execution/metrics.go

```diff
@@ -0,0 +1,56 @@
+package executionsdk
+
+import (
+	"github.com/go-kit/kit/metrics"
+	prometheus "github.com/go-kit/kit/metrics/prometheus"
+	stdprometheus "github.com/prometheus/client_golang/prometheus"
+)
+
+var m *metric
+
+type metric struct {
+	Created    metrics.Counter
```

Comment:
We should prefix with the name of the method:
```suggestion
	CreateCreated    metrics.Counter
```

Reply:
they are already prefixed with the namespace and subsystem so they will have a label like `mesg_execution_created`. These are metrics about the execution itself, not the function of the sdk.

Not so sure why you want to prefix

Reply:
It's only for Go style. These metrics are only used for the creation of an execution. How to name them when we add metric about the update of execution?
It's not really important as it's only a before compilation issue.

Reply:
Not these stats are for the whole lifecycle of the execution:
- Created
- PreSigned (not really useful, mostly for tests)
- Signed
- InProgress
- Completed

That's all the status of the execution from created to completed